web/src/environments/environment.ts

```diff
@@ -0,0 +1,27 @@
+// This file can be replaced during build by using the `fileReplacements` array.
+// `ng build` replaces `environment.ts` with `environment.prod.ts`.
+// The list of file replacements can be found in `angular.json`.
+
+export const environment = {
```

Comment:
https://stackoverflow.com/questions/37482366/is-it-safe-to-expose-firebase-apikey-to-the-public

Reply:
Même si c'est pas un enjeux de sécurité, la bonne pratique serait de ne pas rendre publique ces variables.